packages/@ama-sdk/schematics/README.md

```diff
@@ -79,12 +79,28 @@ If you use `Yarn2+`, you can use the following `scripts` in `package.json`:
 
 Use `generate` to (re)generate your SDK based on the content of `./swagger-spec.yaml` (make sure you have this file at the root of your project) and `upgrade:repository` to regenerate the structure of your project.
 
+#### Light SDK
+
+For SDKs that do not need to handle extensibility, the use of dictionaries or dates, a "light" version of the SDK will be generated. 
```

Comment:
Light sdk is not directly link to dates but more to revivers 
because we could have date but in string